### ISSUE TITLE
required setting is missing: url

### ISSUE THREAD

**Issue by jimmyH** [LukasK32/fleeting-plugin-proxmox#1]
I am probably doing something stupid.

* I can run `make integration-test` successfully.
* I then try and modify an existing runner from fleeting-plugin-aws to fleeting-plugin-proxmox, with the following autoscaler configuration:

```
  [runners.autoscaler]
    capacity_per_instance = 1
    max_use_count = 20
    max_instances = 2
    plugin = "fleeting-plugin-proxmox"
    [runners.autoscaler.plugin_config]
      url = "https://192.168.213.13:8006/"
      insecure_skip_tls_verify = true
      credentials_file_path = "/root/.pve/credentials.json"
      pool = "gitlab"
      storage = ""
      template_id = 132
      max_instances = 2
      instance_network_interface = "eth0"
    [runners.autoscaler.connector_config]
      arch = "amd64"
      protocol = "ssh"
      username = "cloud-user"
      password = ""
      key_path = "/root/.pve/id_rsa"
      use_static_credentials = true
```
* But I get the error `required setting is missing: url` from the plugin, it does not appear to unmarshal the configuration correctly.

I looked at the code and from what little I understand of Go it looks like it should just work. So I'm really hoping I'm just doing something stupid.

I have tried gitlab runners v16.7.1, v17.1.0. Plugin built using `CGO_ENABLED=0 make`:
```
Name:         fleeting-plugin-proxmox
Version:      dev
Git revision: 772db7606865
Git ref:      
GO version:   go1.22.4
Built:        2024-06-07T23:53:20Z
OS/Arch:      linux/amd64
```

**Comment by LukasK32** [comment]
Thanks for reporting!

I reproduced the issue with GitLab Runner and created a patch that fixes it: #2.

Could you test it?

**Comment by jimmyH** [comment]
#2 fixes this issue :)

Very nice plugin. Thanks :)
